README.md

```diff
@@ -2,14 +2,138 @@
+
+```yml
+{app_id}:
+    title: App Title
+    api:
+        versions:
+            - v1
+            - v2
+    channel: '#some-slack-channel'
+    description: App Title is a cool app that does business things for its users.
+    deployment_repo: https://github.com/app-deployment-repo-url
+    disabled_on_prod: true
+    disabled_on_stable: true
+    docs: https://link.to.docs.com/docs
+    frontend:
+        title: App Title Override
+        paths:
+            - /example/path
+            - /another/example/path
+        reload: reload/path
+        sub_apps:
+            -   id: app_id_1
+                title: Some Sub App
+                default: true
+            -   id: app_id_2
+                title: Another Sub App
+        suppress_id: true
+    git_repo: https://github.com/app-development-repo-url
+    mailing_list: app-title@redhat.com
+    top_level: false
+```
+
+### Required Properties (All Apps)
+
+Each of the following properties is required for all apps:
+
+#### app_id
+
+This is your app's ID. It's used as the path to your app, and must be unique.
+
+#### app_id.title
+
+The main title for your app. This is what you want everyone to see when they use your app.
+
+#### app_id.deployment_repo
+
+This is the location of your app's deployment repo (not development repo). These repos generally have `build` or `deploy` as a suffix.
+
+### API Properties
+
+The following properties are used if your app has an API:
+
+#### app_id.api.versions
+
+This is the list of API versions your app can use. Since `v1` is the default, you'll usually want at least that one defined.
+
+### Frontend Properties
+
+The following properties are used if your app has a frontend:
+
+#### app_id.frontend.title
+
+If you want the name of your app to appear differently on the frontend, set this property to override it.
+
+#### app_id.frontend.paths
+
+This is the list of paths where your app will be located. These paths are appended to the end of the paths of its parent apps (if any).
+For example, let's say your app `ex_app_id` is a sub-app of `parent_app`, which has a frontend path of `/parent-app`. If you add a frontend path of `/ex-app`, your app will be available at `/parent-app/ex-app`.
+
+#### app_id.frontend.sub_apps
+
+If your app is a parent to any other apps, those apps should be listed here. 
+Also, if your app has a parent app, or is listed under one of the top-level bundles (e.g. Insights, RHEL, Hybrid), you should add your app to the appropriate sub_apps list.
+
+#### app_id.frontend.reload
+
+This property is almost never needed, and will likely be deprecated. This property is commonly used for Settings apps, and tells Chrome where to navigate on reload.
+
+#### app_id.frontend.suppress_id
+
+This property is almost never needed, and will likely be deprecated. This property is used if the app isn't a real app on disk, and only exists for navigation purposes. 
```

Comment:
Same as above I don't really think we can depracate this because we will probably want some apps to be groupped under one first level nav, but this nav has no app to it (like settings).

Reply:
Yeah - we can't deprecate this now because of the way we're coding it. I feel like there's a way we could get by without making users define this property. For instance, we could probably infer `suppress_id = True` if `deployment_repo` isn't defined for an app.

I don't want to say this definitively until the Chrome nav auto-generation is done, but I still think it's likely we'll get rid of this property at some point

Reply:
I'm removing the part about it rarely being needed and us deprecating it. We can keep that info internal :)